src/integration-test/groovy/au/org/ala/fieldcapture/ManagementUnitSpec.groovy

```diff
@@ -59,28 +60,12 @@ class ManagementUnitSpec extends StubbedCasSpec {
         userId                 | aboutVisible | reportsVisible | sitesVisible | adminVisible
         NO_PERMISSIONS_USER    | true         | false          | false        | false
         EDITOR_USER            | true         | true           | true         | false
+        CASE_MANAGER_USER      | true         | true           | true         | false
```

Comment:
Although the code change passes functional tests, this assertion is actually wrong.  Case manager is the role assigned to grant or project managers and they need to be able to modify permissions.
The reason the functional tests pass is that MERIT requires that any user granted the "case manager" role on a project or management unit also must have the MERIT site/hub level "Officer" role. (What used to be called FC_OFFICER).  
This test needs to be changed so that adminVisible is true.  This can be done by mocking the call that looks up the hub permission to give the user the hub level caseManager role.